src/styles/main.scss

```
@@ -1,3 +1,37 @@
 body {
   margin: 0;
 }
+
+.container {
+  display: grid;
+  grid-template-columns: repeat(auto-fit, minmax(300px, 1fr));
+  justify-content: center;
+
+  &__box {
+    $box-size: 300px;
+    height: $box-size;
+    min-width: $box-size;
+
+    font-family: Arial, sans-serif;
+    font-size: 100px;
+    line-height: $box-size;
+    color: rgb(255, 255, 255);
+    text-align: center;
+  }
+
+  @for $i from 1 through 6 {
+    &__box:nth-child(#{$i})::after {
+      content: "#{$i}";
+    }
+
+    &__box:nth-child(#{$i}) {
+      background-color: rgb(255 - 51 * ($i - 1), 0, 0);
+    }
+  }
+}
+
+@media screen and (min-width: 1200px) {
+  .container {
+    grid-template-columns: repeat(auto-fit, minmax(400px, 1fr));
```

Should be max 3 columns, you have more on wider screen https://prnt.sc/sbced0
```suggestion
    grid-template-columns: repeat(3, 1fr);
```